test/mapKeys.js

```diff
@@ -0,0 +1,9 @@
+var assert = require('assert');
+
+var R = require('..');
+
+describe('mapKeys', function() {
+  it('returns an object with each of the keys passed set to value returned for it by the given function', function() {
+    assert.deepEqual(R.mapKeys(R.toUpper, ['a', 'b', 'c']), {a: 'A', b: 'B', c: 'C'});
+  });
+});
```

Comment:
I think multiset would only matter for a non-deterministic function, not a case I'm particularly worried about, and would happily leave to a last-in-wins scenario.  But I agree that the tests should demonstrate a little more of its behavior, especially around the edges.